metrics.go

```diff
@@ -206,6 +210,14 @@ func (r *RealtimeMetrics) Merge(other *RealtimeMetrics) {
 	for disk, metrics := range other.ByDisk {
 		r.ByDisk[disk] = metrics
 	}
+
+	// merge NetStats
+	if r.ByNet == nil && len(other.ByNet) > 0 {
+		r.ByNet = make(map[string]NetMetrics, len(other.ByNet))
+	}
+	for disk, metrics := range other.ByNet {
```

Comment:
```suggestion
	for nic, metrics := range other.ByNet {
```
Key is not a disk... Is it NIC? From `InterfaceName`?